Imroving the debugging (#108)
0xPolygonMiden/examples: playground/src/markdown/Explainer.md

You can step through the program and see the current VM state displayed in the Output section. 

```
"clk" : 3100
"stack" : [9,31,1,0,1,0,3,1,31,10,50,29,23,19,17,13,11,7,5,3,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
"miden operation" : Some(MovUp3)
"assembly operation" : gt
"assembly num of cycles" : 18
"assembly cycle index" : 13
"memory" : [0]: [2, 0, 0, 0] 
           [1]: [3, 0, 0, 0] 
```

* clk is the current (clock) cycle
* stack shows the current state ot the stack
* miden operation is the current miden vm operation (to which assembler operations get translated)

> * assembly operation shows the currebt Miden assembler operation

Reviewer: typo:
```suggestion
* assembly operation shows the current Miden assembler operation
```

> * assembly num of cycles shows how long the currenct assembly operation takes

Reviewer: typo:
```suggestion
* assembly num of cycles shows how long the current assembly operation takes
```

> * VM Operation is the current miden vm operation (every Operation takes 1 cycle)
> * Assembly Instruction shows the current Miden assembler operation
> * Number of Operations shows cycle count for the instruction
> * Rel. Operation Index tells you at which relative cycle you are in the Instruction

Reviewer: I believe that the assembly <-> operation relation is a bit confusing, and we should try to clarify it a bit better.

I would have the details about an operation together, since they are related. And the instruction first, since that is the thing the user will be changing, it seems to me like it is the most important item. (here and the output too)

In the docs I would also add a link to the Miden documentation. Something like this:

```suggestion
* Assembly Instruction: current Miden assembly instruction.
* Number of Operations: number of operations per instruction.
* Rel. Operation Index: current operation count.
* VM Operation: current Miden VM operation, every operation takes 1 cycle.

Additional documentation on how the VM executes its operations can be foound at the [Miden VM documentation page](https://0xpolygonmiden.github.io/miden-vm/design/programs.html)
```

Author: very good point, changed also the output accordingly
